.travis.yml

```diff
@@ -63,4 +67,19 @@ before_script:
   # Logic for choosing which to use is in the .travis/clone script.
   - .travis/clone "git://github.com/$(dirname $TRAVIS_REPO_SLUG)/htslib.git" "$HTSDIR" "$TRAVIS_BRANCH"
 
-script: autoreconf -i && (cd htslib && autoreconf -i) && (./configure --enable-werror || (cat config.log; /bin/false)) && if test "x$DO_MAINTAINER_CHECKS" = "xyes" ; then make -e maintainer-check ; fi && make -e && make -e test
+script:
+  - |
+    autoreconf -i && \
+    (cd $HTSDIR && \
+      autoreconf -i && \
+      ./configure --prefix=`pwd`/../inst LDFLAGS="$LDFLAGS -Wl,-rpath,`pwd`/../inst/lib" && \
```

Comment:
I don't think you need to set rpath for HTSlib.

Reply:
I do if we want to do a make check inside htslib too, although I since removed that change.  I can cull this again and see if it still works - ought to.